test/e2e/conftest.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+
+import pytest
+import os
+import glob
+
+from pathlib import Path
+
+def pytest_exception_interact(node, call, report):
```

Comment:
This for some reason is not enough to have the Gadgetron logs show up.

Reply:
It looks like the failure is in the call to check the requirements, the results of which are captured for processing instead of being sent to a log file. I'll make sure to resolve this as part of rewriting the requirements code as per the comment below.